care/emr/api/viewsets/report/report_upload.py

```
@@ -0,0 +1,336 @@
+            errors = e.errors()
+            if errors:
+                error = errors[0]
+                error_msg = error.get("msg", str(e))
+                if "Value error," in error_msg:
+                    error_msg = error_msg.replace("Value error, ", "")
+                logger.warning("Validation error for report generation: %s", error_msg)
+                return Response(
+                    {"error": error_msg},
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            logger.exception("Validation error for report generation: %s", e)
```

Ruff (0.14.6)

Redundant exception object included in `logging.exception` call

(TRY401)